File: 3330-find-the-original-typed-string-i/3330-find-the-original-typed-string-i.java
class Solution {
    public int possibleStringCount(String word) {
        int result = 1; 
        int i = 0;
        
        while (i < word.length()) {
            int count = 1;
            while (i + count < word.length() && word.charAt(i) == word.charAt(i + count)) {
                count++;
            }
            if (count > 1) {
                result += count - 1; 
            }
            i += count; 
        }
        
        return result;
    }
}

File: 3330-find-the-original-typed-string-i/3330-find-the-original-typed-string-i.cpp
class Solution {
public:
    int possibleStringCount(string word) {
        int count =0;
        for(int i=1; i<word.size(); i++){
            if(word[i]!=word[i-1]) continue;
            else count++;
        }
        return count+1;
    }
};